master/buildbot/db/builds.py

```diff
@@ -405,8 +405,10 @@ def mkdt(epoch):
 
     @staticmethod
     def _minimal_bdict(row, botmaster):
```

Comment:
Again... 
I think the name `_minimal_bdict` is a little bit outdated. We add to this method more and more fields, so maybe would be a good idea to stop adding new fields or rename this method.

Reply:
For clarity, I've moved back this code to the method.